locales/en/cos-ui.json

```diff
@@ -37,6 +38,12 @@
   "clusterName": "Cluster name",
   "clusterError": "Could not get the cluster name",
   "common": "Common",
+  "communicationErrorDescripton": "Error: \"API message\"",
+  "communicationErrorEmail": "rhosak-eval support@redhat.com",
+  "communicationErrorMessageSubscripton": "To get help, users with a subscription can\n",
```

Comment:
This is the string we would use if we present both options in the message. 
But in a future version when we have just the one message based on their entitlements, we would have a slightly different message that displays:
> To get help, open a support ticket [icon].